File: RemoteSigner/Models/KeyInfo.cs
using System;
namespace RemoteSigner.Models {
    public class KeyInfo {
        public string FingerPrint { get; set; }
        public string Identifier { get; set; }
        public int Bits { get; set; }
        public bool ContainsPrivateKey { get; set; }
        public bool PrivateKeyDecrypted { get; set; }
    }
}

File: pkg/models/KeyInfo.go
package models

type KeyInfo struct {
	FingerPrint           string `example:"0551F452ABE463A4"`
	Identifier            string `example:"Remote Signer Test <test@quan.to>"`
	Bits                  int    `example:"3072"`
	ContainsPrivateKey    bool   `example:"false"`
	PrivateKeyIsDecrypted bool   `example:"false"`
}
